src/scenes/Activities/scenes/EventHistory/index.js

```diff
@@ -40,9 +40,9 @@ class EventHistory extends React.Component {
     super(props);
```

Comment:
We don't need the constructor here either and we can make these methods fat arrow functions and remove the bindings in the constructor.

Reply:
fixed. changed all to fat arrows and moved state outside